plugins/plugin-java/che-plugin-java-ext-lang-server/src/main/java/org/eclipse/che/plugin/java/server/rest/ClasspathService.java

```diff
@@ -19,35 +30,24 @@
 import org.eclipse.jdt.internal.core.JavaModel;
 import org.eclipse.jdt.internal.core.JavaModelManager;
 
-import javax.ws.rs.GET;
-import javax.ws.rs.Path;
-import javax.ws.rs.Produces;
-import javax.ws.rs.QueryParam;
-import javax.ws.rs.core.MediaType;
-import java.util.ArrayList;
-import java.util.List;
-
-import static java.util.Collections.emptyList;
-
 /**
  * Service for getting information about classpath.
  *
  * @author Valeriy Svydenko
  */
 @Path("java/classpath/")
-public class ClasspathService {
+public class ClasspathService implements ClasspathServiceInterface {
 
```

Comment:
Yes, I added this for testing purposes. The fact is that this rest service was not used by sever-side components until now. It was only used through ` ClasspathServiceClient`  which has not exactly the same method signature (it returns a client promise).

So probably there should be a singleton class, say ` ClasspathProvider`  available for injection in server-side components, and used by the `ClasspathService` rest service.

So do you want me to work on removing ` ClasspathServiceInterface` now in this PR, or can it wait for the next PR ?
In other words does it prevent you to approve the merge of this PR or not ?